### ISSUE TITLE
use @chainsafe/ssz 

### ISSUE THREAD

**Issue by GregTheGreek** [ethereum/eth2.0-deposit#61]
You should verify th ssz parameters for the user using th `@chainsafe/ssz`

**Comment by CarlBeek** [comment]
Yeah, the current manual hashing of the SSZ root is a fix to get this off the ground quickly, we fully intend to have these SSZ checks by mainnet.
